server.js

```diff
@@ -2,31 +2,81 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 
+
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
 // 
 // import goldenGlobesData from './data/golden-globes.json'
 // import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
+import booksData from './data/books.json'
+//import netflixData from './data/netflix-titles.json'
 // import topMusicData from './data/top-music.json'
 
 // Defines the port the app will run on. Defaults to 8080, but can be 
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
+const port = process.env.PORT || 8088
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+//Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Endpoints:  /books, /books/:id, /ratings, authors/:author')
 })
 
+//List all the books in the api with /books and query for title
+
+app.get('/books', (req, res) => {
+  const { title } = req.query
+  const bookTitle = booksData.filter((item) =>
+    item.title.toString().toLowerCase().includes(title))
```

Comment:
Careful with capital letters in the title query parameter 🔍

Reply:
I'm not really sure what you mean by this, where is the problem with the capital letters here?